src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -2004,5 +2003,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         FEATURE_BinaryLiterals
         FEATURE_Tuples
         FEATURE_IOperation
+        FEATURE_InterpolatedStrings
```

Comment:
It should be covered by the exist tests. [Test](https://github.com/dotnet/roslyn/blob/master/src/Compilers/VisualBasic/Test/Syntax/Parser/ParseLanguageVersionTests.vb#L304)